subworkflows/nf-core/vcf_annotate_ensemblvep/main.nf

```diff
@@ -7,18 +7,34 @@ include { TABIX_TABIX    } from '../../../modules/nf-core/tabix/tabix/main'
 
 workflow VCF_ANNOTATE_ENSEMBLVEP {
     take:
-    vcf               // channel: [ val(meta), vcf ]
-    fasta             //   value: fasta to use (optionnal)
-    vep_genome        //   value: genome to use
-    vep_species       //   value: species to use
-    vep_cache_version //   value: cache version to use
-    vep_cache         //    path: /path/to/vep/cache (optionnal)
-    vep_extra_files   // channel: [ file1, file2...] (optionnal)
+    ch_vcf                      // channel: [ val(meta), path(vcf) ]
+    ch_fasta                    // channel: [ val(meta2), path(fasta) ] (optional)
+    val_genome                  //   value: genome to use
+    val_species                 //   value: species to use
+    val_cache_version           //   value: cache version to use
+    ch_cache                    // channel: [ val(meta3), path(cache) ] (optional)
+    ch_extra_files              // channel: [ path(file1), path(file2)... ] (optional)
+    ch_custom_extra_files       // channel: [ val(meta), [path(file1), path(file2)...] ] (optional)
```

Comment:
I don't like the ch / val prefix in modules

Reply:
That's a subworkflow ;)